.github/workflows/build-and-test.yml

```diff
@@ -19,12 +19,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Setup dotnet SDK
-        uses: actions/setup-dotnet@v2
+        uses: actions/setup-dotnet@v3
```

Comment:
dependabot update has just merged that will keep these up to date.

Reply:
Might of been me when I was helping fix the action on his computer for 7.0 that I bumped it at the same time...